### DISCUSSION TITLE
nextcloud aio maximum download speed

### DISCUSSION THREAD

**Discussion by Halokk424** [nextcloud/all-in-one#5124] [Questions] [answered]
anyone can relate to their nextcloud aio download speed? i had a gigabit internet 1000up and down. For uploading files to the aio server i can easily get max speed which is around 100MB/s.

But for downloading a file for example a file with few GB. all i get was around 34 MB/s which is only around 1/3 of my up/down speed.

Tried two different machine on my local network running both linux ubuntu and linux mint and both shows the same download speed from the server . Do i need to test them with different network environment instead of local ? or is there something / settings i forgot to set?

Other than that, it runs great without any issue, just this minor problem.
![nextcloud-aio-maximum-download-speed-v0-0ujalwscjlid1](https://github.com/user-attachments/assets/4e2fe45a-781a-4401-b98c-5d5c766f52cd)

**Comment by szaimen** [comment, marked as answer]
This is getting answered at https://www.reddit.com/r/NextCloud/comments/1erwxxg/nextcloud_aio_maximum_download_speed/

**Comment by GuillermoFidalgo** [comment]
The solution has not been posted yet. Does anyone have suggestions to troubleshoot ?
In my case the issue is more drastic.
My uploads are maxing out my connection no matter which client I use (Android or Windows in my case) but downloading from the Windows client (I have the virtual files option enabled) downloads any file of any size at about 13 - 30 KB/s...


<img width="1175" height="474" alt="Screenshot 2025-08-04 192128" src="https://github.com/user-attachments/assets/c1acbd03-5f1d-47ab-8b2e-607945ed4494" />

**Reply by GuillermoFidalgo** [reply]
However downloading from my browser presents no issues. It's only through the client
